lib/koans/03_maps.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Maps do
+  use Koans
+
+  koan "Maps have keys and values" do
+    assert %{ :the_key => "some value" } == %{ :the_key => "some value"}
+  end
+
+  koan "Has keys" do
+    assert Map.keys(%{ :a => "A", :b => "B" }) == [:a, :b]
+  end
+
+  koan "Has values" do
+    assert Map.values(%{ :a => "A", :b => "B" }) == ["A", "B"]
+  end
+
+  koan "Can read values based on a key" do
```

Comment:
I would split the first two into separate koans and possibly leave the `fetch!` version out for now.